test/hummingbot/strategy/cross_exchange_market_making/test_cross_exchange_market_making.py

```diff
@@ -227,13 +239,15 @@ def emit_order_created_event(market: MockPaperExchange, order: LimitOrder):
             )
         )
 
-    def test_both_sides_profitable(self):
+    @async_test(loop=ev_loop)
+    async def test_both_sides_profitable(self):
         self.clock.backtest_til(self.start_timestamp + 5)
-        self.assertEqual(1, len(self.strategy.active_bids))
-        self.assertEqual(1, len(self.strategy.active_asks))
+        await asyncio.sleep(0.5)
```

Comment:
Again it's taken from the AMM Arb. In this particular case it can probably wait for `BuyOrderCreatedEvent` and `SellOrderCreatedEvent`. Should I replace it then also in the AMM Arb?

Reply:
✅